addons/website/static/src/components/fields/fields.js

```diff
@@ -36,6 +38,10 @@ class PageUrlField extends Component {
         this.updateValues();
     }
 
+    get fieldURL() {
+        return (this.props.value.url || this.props.value).replace(/^\//g, '');
```

Comment:
I don't get this fallback on this.props.value it is setting a whole object as url ?
![image](https://user-images.githubusercontent.com/30048408/194782260-76ea9a1e-e8e0-4347-967e-1d0f2cf383ad.png)

- Go to any page
- Empty the URL
- Toggle the redirect dropdown

Reply:
From what I see in the screenshot, you were testing on a '16.0' runbot instance: https://20117364-16-0-all.runbot29.odoo.com/ (not this PR's code :thinking:), but indeed, this flow should be fixed (it leads to a different traceback with the current code too...) fixed now, thanks for spotting this :+1:

Reply:
I was not, I just removed the required=True locally on top of your PR, I couldn't have tested that line on the regular runbot as it doesn't exists :p